Mathlib/Data/Rat/Cardinal.lean

```diff
@@ -0,0 +1,22 @@
+/-
+Copyright (c) 2019 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Floris Van Doorn
+-/
+import Mathlib.Algebra.CharZero.Infinite
+import Mathlib.Algebra.Ring.Rat
+import Mathlib.Data.Rat.Encodable
+import Mathlib.SetTheory.Cardinal.Basic
```

Comment:
Likewise:
```suggestion
import Mathlib.Data.Rat.Encodable
import Mathlib.SetTheory.Cardinal.Basic
```

Reply:
That's not enough imports. I did the `lake exe shake` command as you suggested